src/file-1.xoo

```diff
@@ -0,0 +1,17 @@
+// dummy line 1
+// dummy line 2
+// violate HasTag rule: xoo on line 3
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Has Tag ([xoo:HasTag](http://127.0.0.1:40913/organizations/org0/rules#rule_key=xoo%3AHasTag))

[See it in SonarCloud](http://127.0.0.1:40913/project/issues?id=sample&pullRequest=5537&issues=AWcYyvM02DuvcP4_85m2&open=AWcYyvM02DuvcP4_85m2)

[ISSUE_KEY]: AWcYyvM02DuvcP4_85m2